dags/atd_moped_ecapris_funding_sync.py

```diff
@@ -0,0 +1,126 @@
+# Test locally with: docker compose run --rm airflow-cli dags test atd_moped_ecapris_funding_sync
+
+from os import getenv
+
+from airflow.decorators import dag, task
+from airflow.operators.docker_operator import DockerOperator
+from airflow.models import Param
+from pendulum import datetime, duration
+
+from utils.onepassword import get_env_vars_task
+from utils.slack_operator import task_fail_slack_alert
+
+DEPLOYMENT_ENVIRONMENT = getenv("ENVIRONMENT", "development")
+
+DEFAULT_ARGS = {
+    "owner": "airflow",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 1, 1, tz="America/Chicago"),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+    "retry_delay": duration(minutes=5),
+    "on_failure_callback": task_fail_slack_alert,
+}
+
+REQUIRED_SECRETS = {
+    "SOCRATA_API_KEY_ID": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.apiKeyId",
+    },
+    "SOCRATA_API_KEY_SECRET": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.apiKeySecret",
+    },
+    "SOCRATA_TOKEN": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.appToken",
+    },
+    "SOCRATA_ENDPOINT": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "socrata.endpoint",
+    },
+    "HASURA_ENDPOINT": {
+        "opitem": "Moped Hasura Admin",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Endpoint",
+    },
+    "HASURA_ADMIN_SECRET": {
+        "opitem": "Moped Hasura Admin",
+        "opfield": f"{DEPLOYMENT_ENVIRONMENT}.Admin Secret",
+    },
+    "FUNDING_DATASET_IDENTIFIER": {
+        "opitem": "Moped ETLs",
```

Comment:
this is a new entry in our vault. I followed the pattern of the **Vision Zero ETLs** entry since we've gotten away without needing a Moped ETL-specific entry so far, but now we need one!